resources/testharness.js

```diff
@@ -3671,16 +3692,19 @@ policies and contribution forms [3].
     var tests = new Tests();
 
     if (global_scope.addEventListener) {
-        var error_handler = function(message, stack) {
+        var error_handler = function(error, message, stack) {
```

Comment:
There might be reasonable disagreement to be had about this one, whether an unwrapped `precondition_failed` should be an error or not. And also whether it really makes sense for `allow_uncaught_exception` to allow failing asserts as is currently the case.

https://github.com/web-platform-tests/rfcs/pull/16#issuecomment-550307581